frontend/src/stores.js

```diff
@@ -11,4 +11,4 @@ export const script = writable([]);
  * Points the OFRAK frontend to a seperate backend server
  * When empty, uses the same host and port as the frontend
  */
-export const backendUrl = "";
+export const backendUrl = "//localhost:1666";
```

Comment:
This is probably not supposed to be committed here...

```suggestion
export const backendUrl = "";
```